### DISCUSSION TITLE
Morozov Backpack

### DISCUSSION THREAD

**Discussion by ts-ws** [rjcncpt/StarCitizen-Deutsch-INI#500] [Übersetzungsvorschläge]
Hi,
bei den Morozov Backpack's steht noch "Heavy" in der beschreibung wärend bei allen anderen scheinbar schon "Schwer" steht.

![Morozov Backpack](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/52138452/38708fda-98a3-48f7-a36c-15166dcebc90)

**Comment by MaxM1211** [comment]
Hey 
Danke, waren tatsächlich noch ein paar Zeilen noch nicht übersetzt. 
#501 & #502
